### PR TITLE
creators section

### PR DESCRIPTION
**Pull request by Anvar2726**
image
image

### CODE REVIEW

**Comment by samandarwebcreator** [comment]
Phone responsive qani

**Comment by shamsiddinprogrammer** [comment]
zo'r

File: index.html [hunk 1]
```diff
@@ -6,5 +6,139 @@
...
+              <div class="creators__card-index"><p>12</p></div>
+              <img class="creators__card-img" src="/images/creators12.png" alt="Keepitreal" width="120" height="120">
+              <div>
+                <p class="creators__card-title">Ghiblier</p>
+                <p class="creators__card-text">Total Sales: <span class="creators__card-price">34.53 ETH</span></p>
+              </div>
+            </li>
+
+          </ul>
+          <!-- <a class="creators__link2" href="#">
```
Comment (Muhammadbobur3) [inline index.html:136]: remove comment

File: scss/_general.scss [hunk 2]
```diff
@@ -10,7 +12,7 @@ body {
   min-height: 100vh;
   display: flex;
   flex-direction: column;
-  background-color: #010101;
+  // background-color: #010101;
```
Comment (Muhammadbobur3) [inline scss/_general.scss:15]: remove

File: scss/blocks/_creators.scss [hunk 3]
```diff
@@ -0,0 +1,194 @@
...
+        color: #FFF;
+        font-weight: 600;
+        line-height: 140%;
+        img {
+            vertical-align: middle;
+            margin-right: 12px;
+        }
+    }
+
+    // &__link2{
```
Comment (Muhammadbobur3) [inline scss/blocks/_creators.scss:40]: remove comment

**Review by Muhammadbobur3 — CHANGES_REQUESTED** [review]

**Pull request merged**